### PR TITLE
Update setVisibleListentries_ms.php

### PR DESCRIPTION
**Pull request by a22iliru**
Fixed the query and error in spelling

### CODE REVIEW

**Comment by a22marca** [comment]
The changed code looks good. I just want to make sure that this works properly now. Did you test this on the website in some way @a22iliru ? I will see if I can find a way to test this microservice on the website mean while.

**Comment by a22iliru** [comment]
> The changed code looks good. I just want to make sure that this works properly now. Did you test this on the website in some way @a22iliru ? I will see if I can find a way to test this microservice on the website mean while.

@a22marca I'm not really sure on how to test this

File: DuggaSys/microservices/sectionedService/setVisibleListentries_ms.php [hunk 1]
```diff
@@ -24,11 +24,13 @@
 if (checklogin() && isSuperUser(getUid())){
 
     // prepare SQL query
-    $query = $pdo->prepare("UPDATE listentries Set visibility = :listentryId");
+    $query = $pdo->prepare("UPDATE listentries SET visibility = :visibility WHERE listentryId = :listentryId");
```
Comment (a22marca) [inline DuggaSys/microservices/sectionedService/setVisibleListentries_ms.php:27]: the columns are not correctly named
image

I presume it is the `lid` and `visible` columns that should be used here

**Review by a22marca — CHANGES_REQUESTED** [review]

**Review by a22marca — APPROVED** [review]
OK

**Pull request merged**